dev_guide/builds.adoc

```diff
@@ -549,189 +615,489 @@ source:
 
 // end::using-a-proxy-for-git-cloning-1[]
 
-[[starting-a-build]]
 
-== Starting a Build
-Manually invoke a build using the following command:
+[#using-private-repositories-for-builds]
+=== Using Private Repositories for Builds
 
-----
-$ oc start-build <BuildConfigName>
-----
+Supply valid credentials to build an application from a private repository.
 
-Re-run a build using the `--from-build` flag:
+Currently two types of authentication are supported: basic username-password
+and SSH key based authentication.
 
-----
-$ oc start-build --from-build=<buildName>
-----
+[[basic-authentication]]
+==== Basic Authentication
 
-Specify the `--follow` flag to stream the build's logs in stdout:
+Basic authentication requires either a combination of `username` and `password`,
+or a `token` to authenticate against the SCM server. A `CA certificate` file,
+or a `.gitconfig` file can be attached.
```

Comment:
The terms `username`, `password` and `token` are option names, right?  Suggest: "...of `username` and `password` options, or a `token` option to ..."

For "CA certificate", markup should be single asterisk.